scholia/app/templates/base.html

```diff
@@ -59,8 +59,9 @@
      
     $.getJSON(url, function (data) {
 	 var item = data.entities["{{ q }}"];
+   var title = item.labels.en.value
```

Comment:
This does not look good: the `if` in the next line is there before `item.labels.en` can be non-existing.

Reply:
oh yep of course